DemoApp/README.md

```diff
@@ -0,0 +1,38 @@
+# How to build and test the DemoApp
+
+## Prerequisites
+
+Since `yarn link/unlink` does not work with react-native. Install `yalc` globally. This is a tool that allows you to test local packages without publishing them to npm.
```

Comment:
```suggestion
Since `yarn link/unlink` does not work with react-native. Install [yalc](https://www.npmjs.com/package/yalc) globally. This is a tool that allows you to test local packages without publishing them to npm.
```